File: usuarios/rrhh-usuarios/src/main/java/com/kreitek/rrhh/usuarios/domain/Usuario.java
package com.kreitek.rrhh.usuarios.domain;


import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;

@Entity
public class Usuario {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
        private Integer id;
        private String nombre;
        private String apellidos;
        private String email;
        private String rol;
         private String accion;



        public Integer getId() {
            return id;
        }

        public void setId(Integer id) {
            this.id = id;
        }

        public String getNombre() {
            return nombre;
        }

        public void setNombre(String nombre) {
            this.nombre = nombre;
        }

        public String getApellidos() {
            return apellidos;
        }

        public void setApellidos(String apellidos) {
            this.apellidos = apellidos;
        }

        public String getemail() {
            return email;
        }

        public void setemail(String email) {
            this.email = email;
        }

        public String getrol() {
            return rol;
        }

        public void setrol(String rol) {
            this.rol = rol;
        }

          public String getaccion() {
            return accion;
        }

        public void setaccion(String accion) {
            this.accion = accion;
        }


    }

File: usuarios/frontrrhh/rrhhFront/src/app/components/common/usuario.ts
export class Usuario{
    id: number | undefined;
    nombre: string;
    apellidos: string;
    email: string;
    rol: string;
    accion?: string;
    
  constructor(
    id: number | undefined, 
    nombre: string, 
    apellidos: string, 
    email: string, 
    rol: string, 
    accion?: string
) {
    this.id = id
    this.nombre = nombre
    this.apellidos = apellidos
    this.email = email
    this.rol = rol
    this.accion = accion
  }}